src/popup/basics/index.tsx

```diff
@@ -132,8 +122,10 @@ export const FormSubmitButton = ({
 );
 
 export const FormRow = styled.div`
+  position: relative;
   padding: 0.2rem 0;
-  width: 24.5rem;
+  max-width: 24.5rem;
```

Comment:
`width: 24.5rem` was bigger than the width of form components' parent div if it had paddings.